.github/workflows/decidim_ci.yml

```diff
@@ -69,6 +69,9 @@ jobs:
         shell: "bash"
         run: sudo apt install imagemagick
 
+      - name: Copy example config
+        run: cp config/application.example.yml config/application.yml
```

Comment:
why? if you need any environment var, you must add to `env:` section in the file